### ISSUE TITLE
mopidy playback stops at every bufering error

### ISSUE THREAD

**Issue by solsticedhiver** [mopidy/mopidy#2124]
With mopidy 3.4.1 on archlinuxarm, my mopidy log is filled with errors like the following:

```
SystemError: <built-in method get_value of gi.FieldInfo object at 0xb4322ce0> returned a result with an exception set
OverflowError: Python int too large to convert to C long
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/mopidy/audio/actor.py", line 212, in on_message
    elif msg.type == Gst.MessageType.BUFFERING:
```
And at every such error, the playback of the radio audio stream just stops. and does not restart. You have to force mopidy to start the playback again.

This is very annoying. It is happening evey 5 minutes here; I don't know if my wifi sucks, or if it's the rpi3b being slow or somthing else...

gstreamer 1.22.5, python 3.11.5

**Comment by kingosticks** [comment]
Could you provide the other details? 64bit?

**Comment by solsticedhiver** [comment]
ah yes sorry, 64 bits aka aarch64.

```
mopidy deps
WARNING  2023-09-20 18:17:36,509 [20991:MainThread] mopidy.config
  Ignoring config section 'spotify' because no matching extension was found
Executable: /usr/bin/mopidy
Platform: Linux-6.1.53-1-rpi-ARCH-armv7l-with-glibc2.35
Python: CPython 3.11.5 from /usr/lib/python3.11
Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
  Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
  requests: 2.28.2 from /usr/lib/python3.11/site-packages
    idna: 3.4 from /usr/lib/python3.11/site-packages
    urllib3: 1.26.15 from /usr/lib/python3.11/site-packages
  setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
    platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
    jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
      jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
      autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
      inflect: 7.0.0 from /usr/lib/python3.11/site-packages
        pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
          annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
          pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
    packaging: 23.1 from /usr/lib/python3.11/site-packages
    tomli: 2.0.1 from /usr/lib/python3.11/site-packages
    validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
      fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
  tornado: 6.2 from /usr/lib/python3.11/site-packages
Mopidy-Podcast: 3.0.0 from /usr/lib/python3.11/site-packages
  Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
  Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
  cachetools: 5.3.1 from /usr/lib/python3.11/site-packages
  setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
    platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
    jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
      jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
      autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
      inflect: 7.0.0 from /usr/lib/python3.11/site-packages
        pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
          annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
          pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
    packaging: 23.1 from /usr/lib/python3.11/site-packages
    tomli: 2.0.1 from /usr/lib/python3.11/site-packages
    validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
      fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
  uritools: 4.0.1 from /usr/lib/python3.11/site-packages
Mopidy-TuneIn: 1.1.0 from /usr/lib/python3.11/site-packages
  Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
  Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
  requests: 2.28.2 from /usr/lib/python3.11/site-packages
    idna: 3.4 from /usr/lib/python3.11/site-packages
    urllib3: 1.26.15 from /usr/lib/python3.11/site-packages
  setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
    platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
    jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
      jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
      autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
      inflect: 7.0.0 from /usr/lib/python3.11/site-packages
        pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
          annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
          pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
    packaging: 23.1 from /usr/lib/python3.11/site-packages
    tomli: 2.0.1 from /usr/lib/python3.11/site-packages
    validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
      fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
Mopidy-RadioNet: 0.2.2 from /usr/lib/python3.11/site-packages
  Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
  Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
  setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
    platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
    jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
      jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
      autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
      inflect: 7.0.0 from /usr/lib/python3.11/site-packages
        pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
          annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
          pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
    packaging: 23.1 from /usr/lib/python3.11/site-packages
    tomli: 2.0.1 from /usr/lib/python3.11/site-packages
    validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
      fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
  uritools: 4.0.1 from /usr/lib/python3.11/site-packages
Mopidy-MusicBox-Webclient: 3.1.0 from /usr/lib/python3.11/site-packages
  Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
  Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
  setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
    platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
    jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
      jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
      autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
      inflect: 7.0.0 from /usr/lib/python3.11/site-packages
        pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
          annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
          pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
    packaging: 23.1 from /usr/lib/python3.11/site-packages
    tomli: 2.0.1 from /usr/lib/python3.11/site-packages
    validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
      fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
Mopidy-SomaFM: 2.0.2 from /usr/lib/python3.11/site-packages
  Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
  Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
  requests: 2.28.2 from /usr/lib/python3.11/site-packages
    idna: 3.4 from /usr/lib/python3.11/site-packages
    urllib3: 1.26.15 from /usr/lib/python3.11/site-packages
  setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
    platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
    jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
      jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
      autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
      inflect: 7.0.0 from /usr/lib/python3.11/site-packages
        pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
          annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
          pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
    packaging: 23.1 from /usr/lib/python3.11/site-packages
    tomli: 2.0.1 from /usr/lib/python3.11/site-packages
    validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
      fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
Mopidy-Podcast-iTunes: 3.0.1 from /usr/lib/python3.11/site-packages
  Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
  Mopidy-Podcast: 3.0.0 from /usr/lib/python3.11/site-packages
    Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
    Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
    cachetools: 5.3.1 from /usr/lib/python3.11/site-packages
    setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
      platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
      jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
        jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
          more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
        jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
        autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
        inflect: 7.0.0 from /usr/lib/python3.11/site-packages
          pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
            annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
            pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
              typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
      packaging: 23.1 from /usr/lib/python3.11/site-packages
      tomli: 2.0.1 from /usr/lib/python3.11/site-packages
      validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
        fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
    uritools: 4.0.1 from /usr/lib/python3.11/site-packages
  Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
  requests: 2.28.2 from /usr/lib/python3.11/site-packages
    idna: 3.4 from /usr/lib/python3.11/site-packages
    urllib3: 1.26.15 from /usr/lib/python3.11/site-packages
  setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
    platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
    jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
      jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
      autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
      inflect: 7.0.0 from /usr/lib/python3.11/site-packages
        pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
          annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
          pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
    packaging: 23.1 from /usr/lib/python3.11/site-packages
    tomli: 2.0.1 from /usr/lib/python3.11/site-packages
    validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
      fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
Mopidy-ALSAMixer: 2.0.1 from /usr/lib/python3.11/site-packages
  Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
  Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
  setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
    platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
    jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
      jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
      autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
      inflect: 7.0.0 from /usr/lib/python3.11/site-packages
        pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
          annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
          pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
    packaging: 23.1 from /usr/lib/python3.11/site-packages
    tomli: 2.0.1 from /usr/lib/python3.11/site-packages
    validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
      fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
  pyalsaaudio: 0.10.0 from /usr/lib/python3.11/site-packages
Mopidy-SoundCloud: 3.0.2 from /usr/lib/python3.11/site-packages
  Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
  Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
  setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
    platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
    jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
      jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
      autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
      inflect: 7.0.0 from /usr/lib/python3.11/site-packages
        pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
          annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
          pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
    packaging: 23.1 from /usr/lib/python3.11/site-packages
    tomli: 2.0.1 from /usr/lib/python3.11/site-packages
    validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
      fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
  requests: 2.28.2 from /usr/lib/python3.11/site-packages
    idna: 3.4 from /usr/lib/python3.11/site-packages
    urllib3: 1.26.15 from /usr/lib/python3.11/site-packages
  beautifulsoup4: 4.12.2 from /usr/lib/python3.11/site-packages
    soupsieve: 2.5 from /usr/lib/python3.11/site-packages
Mopidy-YouTube: 3.6 from /usr/lib/python3.11/site-packages
  beautifulsoup4: 4.12.2 from /usr/lib/python3.11/site-packages
    soupsieve: 2.5 from /usr/lib/python3.11/site-packages
  cachetools: 5.3.1 from /usr/lib/python3.11/site-packages
  Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
  Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
  requests: 2.28.2 from /usr/lib/python3.11/site-packages
    idna: 3.4 from /usr/lib/python3.11/site-packages
    urllib3: 1.26.15 from /usr/lib/python3.11/site-packages
  setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
    platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
    jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
      jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
      autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
      inflect: 7.0.0 from /usr/lib/python3.11/site-packages
        pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
          annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
          pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
    packaging: 23.1 from /usr/lib/python3.11/site-packages
    tomli: 2.0.1 from /usr/lib/python3.11/site-packages
    validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
      fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
Mopidy-MPD: 3.3.0 from /usr/lib/python3.11/site-packages
  Mopidy: 3.4.1 from /usr/lib/python3.11/site-packages
  Pykka: 3.1.1 from /usr/lib/python3.11/site-packages
  setuptools: 68.0.0 from /usr/lib/python3.11/site-packages
    platformdirs: 3.10.0 from /usr/lib/python3.11/site-packages
    jaraco.text: 3.11.1 from /usr/lib/python3.11/site-packages
      jaraco.functools: 3.9.0 from /usr/lib/python3.11/site-packages
        more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
      jaraco.context: 4.3.0 from /usr/lib/python3.11/site-packages
      autocommand: 2.2.2 from /usr/lib/python3.11/site-packages
      inflect: 7.0.0 from /usr/lib/python3.11/site-packages
        pydantic: 2.3.0 from /usr/lib/python3.11/site-packages
          annotated-types: 0.5.0 from /usr/lib/python3.11/site-packages
          pydantic-core: 2.6.3 from /usr/lib/python3.11/site-packages
            typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
          typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
        typing-extensions: 4.8.0 from /usr/lib/python3.11/site-packages
      more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    more-itertools: 10.1.0 from /usr/lib/python3.11/site-packages
    ordered-set: 4.1.0 from /usr/lib/python3.11/site-packages
    packaging: 23.1 from /usr/lib/python3.11/site-packages
    tomli: 2.0.1 from /usr/lib/python3.11/site-packages
    validate-pyproject: 0.13.post1.dev0+gb752273.d20230605 from /usr/lib/python3.11/site-packages
      fastjsonschema: 2.18.0 from /usr/lib/python3.11/site-packages
GStreamer: 1.22.5.0 from /usr/lib/python3.11/site-packages/gi
  Detailed information: 
    Python wrapper: python-gi 3.46.0
    Relevant elements:
      Found:
        uridecodebin
        souphttpsrc
        appsrc
        alsasink
        osssink
        oss4sink
        pulsesink
        id3demux
        id3v2mux
        lamemp3enc
        mpegaudioparse
        mpg123audiodec
        vorbisdec
        vorbisenc
        vorbisparse
        oggdemux
        oggmux
        oggparse
        flacdec
        flacparse
        shout2send
      Not found:
        flump3dec
        mad
```

**Comment by solsticedhiver** [comment]
I filled this issue out of frustration, but may be I was wrong to think it was mopidy;

I changed my AP settings by disabling the option "Disassociate On Low Acknowledgement condition" and it seems better, because the rpi is not disconnected anymore it seems, and does not take ages to reconnect.

I am still having the errors in the log, but without playback stopping anymore.

Sorry. I am closing this issue.
